src/v2/api/responses/AllEventResponse.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { DisciplineTypeEnum } from '@prisma/client';
+
+export class AllEventResponse {
+  @ApiProperty()
+    id: string;
+
+  @ApiProperty()
+    name: string;
+
+  @ApiProperty()
+    startTime: Date;
+
+  @ApiProperty()
+    endTime: Date;
+
+  @ApiPropertyOptional({
```

Comment:
It's not optional. Optional means undefined. This field can be type or null, not undefined